middlewares/index.mjs

```diff
@@ -0,0 +1,12 @@
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+
+const middlewares = [
+  bodyParser.urlencoded({ extended: true }),
+  bodyParser.json(),
+  cookieParser('Node js mentoring programm'),

```

Comment:
In addition: cookie and query should be added to request stream object as parsedCookies and parsedQuery respectively
- Item **6** and **7** of the task

Reply:
We have req.query and req.cookies. What difference other than 'parsed' prefix?)

Reply:
Agreed. From my point of view there is no other difference as well